src/PerfView/CommandLineArgs.cs

```diff
@@ -462,7 +462,7 @@ private void SetupCommandLine(CommandLineParser parser)
                 // TODO this logic is cloned.  We need it in only one place.  If you update it do the other location as well
                 // The process events are so we get process names.  The ImageLoad events are so that we get version information about the DLLs 
                 KernelEvents = KernelTraceEventParser.Keywords.Process | KernelTraceEventParser.Keywords.ImageLoad;
-                ClrEvents = ClrTraceEventParser.Keywords.GC | ClrTraceEventParser.Keywords.Exception;
+                ClrEvents = ClrTraceEventParser.Keywords.GC | ClrTraceEventParser.Keywords.Type | ClrTraceEventParser.Keywords.GCHeapAndTypeNames | ClrTraceEventParser.Keywords.Exception;
```

Comment:
Having this for GCOnly makes sense, but I don't think we'd want this for GCCollectOnly.  @Maoni0, what do you think?

Reply:
we definitely wouldn't want this in GCCollectOnly. GCCollectOnly is supposed to be very lightweight and can be on all the time. GCHeapAndTypeNames is extremely heavy weight and artificially increases the GC pause time by a lot so should not be on for GCCollectOnly. I don't think it should be on for GCOnly either 'cause it instantly changes the perf characteristics of GC but since I'm not aware anyone actually uses GCOnly I don't have a strong opinion there.

Reply:
Thanks for clarifying @Maoni0.  Based on this, I think that we should not make this change.  Is the type information just missing for /GCCollectOnly and /GCOnly?  Specifically, does the type information appear when using the default collection parameters?

Reply:
I never use /GCCollectOnly so I don't have an opinion about the behavior there. I frequently use /GCOnly, which is where the notable functionality gap was observed (/GCOnly provides information for the GC Stats view, except the finalized objects information is missing).

Reply:
which finalizer object info? we fire this event for finalized objects:
```
                    <event value="29" version="0" level="win:Verbose"  template="FinalizeObject"
                           keywords ="GCKeyword"
                           opcode="FinalizeObject"
                           task="GarbageCollection"
                           symbol="FinalizeObject" message="$(string.RuntimePublisher.FinalizeObjectEventMessage)"/>
```
and that's on the GC keyword verbose level

Reply:
@Maoni0 either that event is not sufficient, or PerfView does not know how to interpret it. Here is the GCStats prior to this change:

![image](https://user-images.githubusercontent.com/1408396/72283065-d7f90380-35f2-11ea-8712-88c5f77cfde2.png)

Here it is after this change: